src/components/pages/HomePage.js

```diff
@@ -0,0 +1,311 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Menu, Modal, Input, Grid, Button, Form } from 'semantic-ui-react';
+import { Link } from 'react-router-dom'
+
+import requestPipeline from '../../components/pages/requestPipeline';
+
+import Styles from '../../styles/HomePage';
+import Actions from '../../reducers/Actions'
+import { statekeys } from '../../helpers/Common'
+
+export class HomePage extends Component {
+  render() {
+    const {setEmail, setPassword, setFirstName, setLastName, setPasswordRepeat, setAddress, setCity, setAddressState, setZipcode, openLoginModal, closeLoginModal, isLoginModalOpen, openSignupModal, closeSignupModal, isSignupModalOpen} = this.props;
```

Comment:
You can put each of these onto their own line. Something like:

```
const {
    foo,
    bar,
    ...
} = this.props;
```

That will help make that line more readable.